src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,10 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} />
```

Comment:
Consider adding a `key` prop to the `CommentInfo` component inside the `map` function. This helps React identify which items have changed, are added, or are removed, and is necessary for optimal rendering performance. Typically, you would use a unique identifier from the `comment` object, such as `comment.id`.